dlk/python/dlk/templates/include/memdriver.h

```diff
@@ -21,11 +21,55 @@ limitations under the License.
 #include <stdint.h>
 #include <fcntl.h>
 #include <unistd.h>
+#include <memory>
+#include <system_error>
+
+class FDManager {
+ public:
+  FDManager() : fd(-1) {}
+  FDManager(int fd) : fd(fd) {}
+  ~FDManager() {
+    if (fd >= 0) {
+      close(fd);
+    }
+  }
+  operator int() const { return fd; }
+ private:
+  int fd;
+};
+
+class SimpleMappedMem {
```

Comment:
Since `MappedMem` class seems not be used in the current code, how about removing current `MappedMem` class and renaming this `SimpleMappedMem` class to `MappedMem` class?

Reply:
Fixed.